docs/api/python/getting-started.md

```diff
@@ -207,324 +369,882 @@ if __name__ == "__main__":
+    """
+    Execute finalize withdraw transaction on L1 network
+    :type zksync_provider:
+         Instance of ZkSync provider
+    :param ethereum_provider
+        Instance of EthereumProvider
+    :param withdraw_tx_hash
+        Hash of withdraw transaction on L2 network
+    :return:
+        TxReceipt of finalize withdraw transaction on L1 network
+    """
+    zks_receipt = zksync_provider.zksync.wait_finalized(withdraw_tx_hash)
+
+    # Check if withdraw transaction was successful
+    if not zks_receipt["status"]:
+        raise RuntimeError("Withdraw transaction on L2 network failed")
+
+    # Execute finalize withdraw
+    tx_receipt = ethereum_provider.finalize_withdrawal(zks_receipt["transactionHash"])
+
+    # Check if finalize withdraw transaction was successful
+    if not tx_receipt["status"]:
+        raise RuntimeError("Finalize withdraw transaction L1 network failed")
+    return tx_receipt
 
 
-def withdraw():
-    value = Decimal("0.001")
+if __name__ == "__main__":
+    # Get the private key from OS environment variables
+    PRIVATE_KEY = bytes.fromhex(os.environ.get("PRIVATE_KEY"))
+    WITHDRAW_TX_HASH = HexBytes.fromhex(os.environ.get("WITHDRAW_TX_HASH"))
 
-    account: LocalAccount = Account.from_key("PRIVATE_KEY")
-    zksync_web3 = ZkSyncBuilder.build("ZKSYNC_NETWORK_URL")
-    chain_id = zksync_web3.zksync.chain_id
-    signer = PrivateKeyEthSigner(account, chain_id)
-    ETH_TOKEN = Token.create_eth()
+    # Set a provider
+    ZKSYNC_PROVIDER = "https://zksync2-testnet.zksync.dev"
+    ETH_PROVIDER = "https://rpc.ankr.com/eth_goerli"
 
-    nonce = zksync_web3.zksync.get_transaction_count(account.address, ZkBlockParams.COMMITTED.value)
-    gas_price = zksync_web3.zksync.gas_price
-    bridges: BridgeAddresses = zksync_web3.zksync.zks_get_bridge_contracts()
+    # Connect to zkSync network
+    zk_web3 = ZkSyncBuilder.build(ZKSYNC_PROVIDER)
 
-    l2_func_encoder = L2BridgeEncoder(zksync_web3)
-    call_data = l2_func_encoder.encode_function(fn_name="withdraw", args=[
-        account.address,
-        ETH_TOKEN.l2_address,
-        ETH_TOKEN.to_int(value)
-    ])
+    # Connect to Ethereum network
+    eth_web3 = Web3(Web3.HTTPProvider(ETH_PROVIDER))
+    eth_web3.middleware_onion.inject(geth_poa_middleware, layer=0)
 
-    func_call = TxFunctionCall(chain_id=chain_id,
-                               nonce=nonce,
-                               from_=account.address,
-                               to=bridges.l2_eth_default_bridge,
-                               data=HexStr(call_data),
-                               gas_limit=0, # unknown at this state, will be replaced by estimate_gas
-                               gas_price=gas_price,
-                               max_priority_fee_per_gas=100000000)
-    estimate_gas = zksync_web3.zksync.eth_estimate_gas(func_call.tx)
-    print(f"Fee for transaction is: {estimate_gas * gas_price}")
+    # Get account object by providing from private key
```

Comment:
```suggestion
    # Get account object by providing private key of the sender?
```
I don't know if you mean this but "providing from private key" is misleading

Reply:
`Get account object by providing private key of the sender` is a correct.

Reply:
Thank you :)